inference/inference.py

```diff
@@ -133,8 +133,9 @@ def setup(params):
     else:
       params['N_in_channels'] = n_in_channels
     params['N_out_channels'] = n_out_channels
-    params.means = np.load(params.global_means_path)[0, out_channels] # needed to standardize wind data
-    params.stds = np.load(params.global_stds_path)[0, out_channels]
+    params.means = valid_dataset.out_means[0] # needed to standardize wind data
```

Comment:
This is tangential to this PR but I'm not sure what the dims of `params.means` winds up being in the end. Feel free to ignore.

Reply:
It's a scalar for each variable, although some reshaping (e.g. see FV3GFS data loader) is done to allow operation with tensors that include batch/x/y dims.